fast-dump/src/main/cpp/klog.h

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020 Kwai, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Author: Qiushi Xue <xueqiushi@kuaishou.com>
```

Comment:
We generally don't maintain author names in our repositories, unless it's a copy paste from AOSP. Author names are preserved by git history.

Reply:
done